### PR TITLE
Resurrect owamp from boneyard by including the missing patch inline

### PR DESCRIPTION
**Pull request by candlerb**
See #43361 and Homebrew/homebrew-boneyard#66

### CODE REVIEW

**Comment by bfontaine** [comment]
Please also edit `tap_migrations.rb` to remove this line: https://github.com/Homebrew/homebrew/blob/51be1a80a8dfadfdc3b91e5dc51f16512dd9fc3c/Library/Homebrew/tap_migrations.rb#L160

**Comment by bfontaine** [comment]
How were you able to retrieve the missing patch? :open_mouth:

**Comment by DomT4** [comment]
This will need to meet the standards in `brew audit --strict --online` now, as it'll be treated like a new formula by the CI.

**Comment by candlerb** [comment]
> How were you able to retrieve the missing patch? :open_mouth:

From `/Library/Caches/Homebrew/owamp--patch-9e5588d57b357f438ae1a785a713f0deaea5a5ba.patch` on a machine where I'd built it previously

**Comment by candlerb** [comment]
> This will need to meet the standards in brew audit --strict --online now, as it'll be treated like a new formula by the CI.

Done, except I don't know how to get rid of the following error:

~~~
==> audit problems
owamp:
 * Formula name conflicts with homebrew/boneyard/owamp
~~~

This is after editing `tap_migrations.rb` as per @bfontaine

**Comment by DomT4** [comment]
You won't be able to get rid of that until it is removed from the boneyard. If that's the only failure, we can ignore it.

**Comment by candlerb** [comment]
It's the only error now. I fixed two others:

~~~
 * Stable: SHA1 checksums are deprecated, please use SHA256
 * `require "formula"` is now unnecessary
~~~

File: Library/Formula/owamp.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class Owamp < Formula
+  desc "Implementation of the One-Way Active Measurement Protocol"
+  homepage "http://www.internet2.edu/performance/owamp/"
```
Comment (DomT4) [inline Library/Formula/owamp.rb]: It looks like this more or less redirects to `http://software.internet2.edu/owamp/` now, so perhaps worth using that?

**Comment by DomT4** [comment]
Changed my mind on the notation. Fixed it after merge rather than redoing the whole CI dance.

Merged in https://github.com/Homebrew/homebrew/commit/0d4bfc41df578b66879a921951bb0efca6be43dd. Thank you for your contribution to Homebrew @candlerb! :smiley_cat:

**Pull request closed without merging**